src/Silk.Core.Logic/Commands/Bot/ShardsCommand.cs

```diff
@@ -41,9 +42,9 @@ public async Task Shards(CommandContext ctx)
             sb.Append('|');
             sb.Append($"{averagePing / shards.Count}".Center("\t[Ping]\t"));
             sb.Append('|');
-            sb.Append($"{Discord.Bot.Instance!.Client.ShardClients.SelectMany(c => c.Value.Guilds.Keys).Count()}".Center("\t[Guilds]\t"));
+            sb.Append($"{shards.SelectMany(c => c.Value.Guilds).Count()}".Center("\t[Guilds]\t"));
             sb.Append('|');
-            sb.Append($" {Discord.Bot.Instance!.Client.ShardClients.Values.SelectMany(g => g.Guilds.Values).SelectMany(g => g.Members.Keys).Count()}".Center("\t[Members]\t"));
+            sb.Append($" {shards.Values.SelectMany(g => g.Guilds.Values).SelectMany(g => g.Members).Count()}".Center("\t[Members]\t"));
```

Comment:
See the above review comment. It still applies, it's just a nested LINQ statement instead.